exampleSite/content/jobs/healers/scholar/_index.md

```diff
@@ -1,4 +1,5 @@
 ---
 title: Scholar
```

Comment:
Small nitpick – will we need the title anymore?

Reply:
It's still used in the single_header.html for example. There's probably a way to get from the job_name string to the title and vice versa with some string manipulation. I think @Roxas92 did something like that in his PR.